source/MaterialXTest/MaterialXRender/RenderUtil.h

```diff
@@ -128,6 +128,9 @@ class ShaderRenderTester
     // Code generation methods
     //
 
+    // Register any additional source code paths used by the generator
+    virtual void registerSourceCodeSearchPaths(mx::GenContext& /*context*/) {};
```

Comment:
I believe this method is now unused in MaterialXRender, and we can omit this from your changelist.  Let me know if there are subtle cases I'm missing, though.